console.lua

```diff
@@ -187,6 +187,73 @@ commands = {{
         end
         return "Discards are now " .. G.GAME.current_round.discards_left
     end
+}, {
+    name = "joker",
+    source = "debugplus",
+    shortDesc = "Spawns a joker",
+    desc = [[Spawns Jokers. Usage:\njoker {args} [name]\n 
+            -a : Skip calling add_to_deck on spawned joker
+            -c : Count of jokers to spawn
+            -e : Edition to spawn jokers with
+            -E : enable jokers spawned to have random editions
+        ]],
+    exec = function(args, rawArgs, dp)
+        if G.STAGE ~= G.STAGES.RUN then
+            return "This command must be run during a run.", "ERROR"
+        end
+        local j_key = nil
+        local j_count = 1
+        
+        local j_no_edition = true
+        local j_set = "Joker"
+        local j_edition = nil
+        local j_rarity = nil
+        local add_to_deck = true
+
+        -- I thought of making a full on arg parser for this command
```

Comment:
I maybe should make an arg parser for more complicated commands like this. It wouldn't need to be blocking this, but I would hate for every command with flags to need to reimplement this.

Reply:
There are libraries out there that do this far better than I did.
It can be generalized but I normally find it's easier to take a rough frame work and just fill it in